Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ however, it has to be formatted properly to pass verification tests.
 ## [Unreleased]
 
 ### Fixed
+* Fixed an issue where a layout-override registered via `InputSystem.RegisterLayoutOverride(...)` would cause the editor to malfunction or crash if the layout override had a name already used by an existing layout. (case 1377685).
```

Comment:
Nitpick: empty line between header and start of list.

Reply:
For case, link to case on issue tracker pls.

Reply:
It was reported internally so there is unfortunately no issue tracker link